### DISCUSSION TITLE
Feature Request - Export as PNG in Controls

### DISCUSSION THREAD

**Discussion by pgupta56** [xyflow/xyflow#1756] [New Features]
Is it possible to add a new option in controls in V10?

**Comment by moklick** [comment]
You can already do it by passing a child node to the controls.

**Reply by pgupta56** [reply]
ohh cool !! Thanks !!

**Reply by moklick** [reply]
I was just speaking about adding another button to the controls. The png export is not on our list.

**Comment by moklick** [comment]
We will not integrate it as a feature but I've created a ["Download Image" example](https://reactflow.dev/docs/examples/misc/download-image/) that uses [html-to-image](https://github.com/bubkoo/html-to-image).
